wasmtime-jit fails when built individually

### Environment info
- `rustc 1.34.0 (91856ed52 2019-04-10)`
- `Linux 4.9.0-8-amd64 #1 SMP Debian 4.9.144-3 (2019-02-02) x86_64 GNU/Linux`
- on master branch, commit `3351bef`

### Repro
`cd wasmtime-jit && cargo build --features default`

### Output
```
error[E0432]: unresolved import `std::ops`
 --> wasmtime-environ/src/compilation.rs:9:10
  |
9 | use std::ops::Range;
  |          ^^^ did you mean `rayon::std::ops`?

error[E0554]: #![feature] may not be used on the stable release channel
  --> wasmtime-environ/src/lib.rs:28:35
   |
28 | #![cfg_attr(not(feature = "std"), feature(alloc))]
   |                                   ^^^^^^^^^^^^^^

error[E0282]: type annotations needed
  --> wasmtime-environ/src/compilation.rs:32:30
   |
32 |                 .map(|range| buffer[range].to_vec())
   |                              ^^^^^^^^^^^^^ cannot infer type
   |
   = note: type must be known at this point

error: aborting due to 3 previous errors

Some errors occurred: E0282, E0432, E0554.
For more information about an error, try `rustc --explain E0282`.
error: Could not compile `wasmtime-environ`.
```
- Interestingly, wasmtime-environ does not fail when built individually.
- Building on nightly (`rustc 1.36.0-nightly (3c3d3c177 2019-04-17)`) obviously gets rid of the error relating to the feature attribute, but retains the same errors about the type inference and resolving `std::ops`.

Perhaps this may be related in some way to #75?

--- a/README.md
+++ b/README.md
@@ -56,4 +56,24 @@ Additional goals for Wasmtime include:
 [Cranelift]: https://github.com/CraneStation/cranelift
 [Lightbeam]: https://github.com/CraneStation/lightbeam
 
+#### Including Wasmtime in your project
+Wasmtime exposes an API for JIT compilation through the `wasmtime-jit` subcrate, which depends on `wasmtime-environ` and `wasmtime-runtime` for the ABI and runtime support respectively. However, this API is not documented and subject to change. Please use at your own risk!
+
+Build the individual crates as such:
+
+```
+cargo build --package wasmtime-jit
+```
+
+Wasmtime does not currently publish these crates on crates.io. They may be included as a git dependency, like this:
+
+```toml
+[dependencies]
+wasmtime-environ = { git = "https://github.com/CraneStation/wasmtime", rev = "somecommithash" }
+wasmtime-runtime = { git = "https://github.com/CraneStation/wasmtime", rev = "somecommithash" }
+wasmtime-jit = { git = "https://github.com/CraneStation/wasmtime", rev = "somecommithash" }
+```
+
+All three crates must be specified as dependencies for `wasmtime-jit` to build correctly, at the moment.
+
 It's Wasmtime.

